dockers/docker-lldp-sv2/Dockerfile.j2

```diff
@@ -35,12 +35,13 @@ RUN apt-get purge -y python-pip && \
            /python-wheels          \
            ~/.cache
 
+COPY ["docker-lldp-init.sh", "/usr/local/bin/"]
```

Comment:
can you please put the script into /usr/bin?
We never use /usr/local for anything (but agree it is more appropriate to put our own code).

Reply:
@pavel-shirshov Done